plugins/woocommerce-blocks/assets/js/blocks/product-filters/utils/get-colors-from-block-supports.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import { BlockAttributes } from '@wordpress/blocks';
+
+function getCSSVar( slug: string | undefined, value: string | undefined ) {
```

Comment:
I believe this is already available here: https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-blocks/assets/js/blocks/product-filters/utils/colors/index.ts#L13